tests/test_dataset.py

```diff
@@ -0,0 +1,47 @@
+import json
+import pytest
+import dask.dataframe as dd
+from pathlib import Path
+
+from fondant.manifest import Manifest
+from fondant.dataset import FondantDataset
+
+manifest_path = Path(__file__).parent / "example_data/manifest.json"
+dataset_path = Path(__file__).parent / "example_data/151"
+
+@pytest.fixture
+def subset_factory(tmp_path_factory):
```

Comment:
Is there a reason to prepare the data dynamically instead of storing it in the expected state?

Reply:
I removed the dynamic part